Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
```

Comment:
Can we use FROM python 3.9 here? I think 2 x FROM are possible

Reply:
Not everyone has or even needs CUDA support... the wonderful thing about docker is that you can mix and match it - why limit the users to one usecase?

Reply:
Thank you for your feedback! I will try to address it in a new pull request.

As for multiple FROM: I know how it works, but it is not applicable in our case. It is only useful for copying a particular set of files from one image to another. For roop we need to combine cuda with python, which are both spread across the filesystem of the Docker image and are hard to cherry pick.